vim25/types/enum.go

```diff
@@ -650,9 +650,11 @@ type DasVmPriority string
 
 const (
 	DasVmPriorityDisabled = DasVmPriority("disabled")
+	DasVmPriorityLowest   = DasVmPriority("lowest")
```

Comment:
The vim25/types package is generated from API's WSDL, so this change would be overwritten.